twentytwentyone-blocks/style.css

```diff
@@ -103,7 +103,7 @@ body {
 
 	.wp-site-blocks .alignfull {
 		transform: translateX(0px);
-		width: 100% + var(--wp--custom--spacing--horizontal));
+		width: calc(100% + var(--wp--custom--spacing--horizontal));
```

Comment:
If I recall from #134, these should just be `100%` right? I don't think the custom spacing was necessary.

Reply:
Yeah that would be better actually, I'll change it...

Reply:
Done